src/Utility.ts

```diff
@@ -92,13 +92,12 @@ export namespace Utility {
     }
 
     export async function validateInstallPath(installPath: string): Promise<boolean> {
-        const serverConfigFile: string = path.join(installPath, 'conf', 'server.xml');
-        const serverWebFile: string = path.join(installPath, 'conf', 'web.xml');
-        const serverBootstrapJarFile: string = path.join(installPath, 'bin', 'bootstrap.jar');
-        const serverJuliJarFile: string = path.join(installPath, 'bin', 'tomcat-juli.jar');
+        const configFileExists: boolean = await fse.pathExists(path.join(installPath, 'conf', 'server.xml'));
+        const serverWebFileExists: boolean = await fse.pathExists(path.join(installPath, 'conf', 'web.xml'));
+        const serverBootstrapJarFileExists: boolean = await fse.pathExists(path.join(installPath, 'bin', 'bootstrap.jar'));
+        const serverJuliJarFileExists: boolean = await fse.pathExists(path.join(installPath, 'bin', 'tomcat-juli.jar'));
 
-        return (!await fse.pathExists(serverConfigFile) || !await fse.pathExists(serverWebFile) ||
-            !await fse.pathExists(serverBootstrapJarFile) || !await fse.pathExists(serverJuliJarFile));
+        return configFileExists && serverWebFileExists && serverBootstrapJarFileExists && serverJuliJarFileExists;
```

Comment:
`const configFileExists: Promise<boolean> = fse.pathExists(path.join(installPath, 'conf', 'server.xml'));
...

return await configFileExists && await serverWebFileExists && ...;
`

Seems this can be more efficient I think.

Reply:
great suggestion.